docs/content/integrations/airlift.mdx

```diff
@@ -23,7 +23,7 @@ Airlift depends on the the availability of Airflow’s REST API. Airflow’s RES
 
 ## Airflow Migration Tutorial
 
-In this tutorial, we'll use `dagster-airlift` to migrate an Airflow dag to Dagster asssets. [Click here to get started](/integrations/airlift/tutorial).
+In this tutorial, we'll use `dagster-airlift` to migrate an Airflow dag to Dagster asssets. [Click here to get started](/integrations/airlift/tutorial/overview).
```

Comment:
```suggestion
In this tutorial, we'll use `dagster-airlift` to migrate an Airflow dag to Dagster assets. [Click here to get started](/integrations/airlift/tutorial/overview).
```

Adding suggestion for `graphite-app` in case it makes it easier to "accept change".